src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccount.fs

```diff
@@ -0,0 +1,69 @@
+﻿module Deciders.BankAccount
+
+open System
+open BankAccountPrimitives
+
+type BankAccountOpened =
+    { BankAccountId: AccountId
```

Comment:
Controversial but (I had to be sold this and hated it for a day or two - I came from C# and had used PascalCase in F# for years)....

I name fields of records with `camelCase`. That's technically a breach of .NET Pascal Case property name violations. Many major companies do it.

It fits well with casing of names of functions and just about everything in general F# code

Also means you dont need JSON serializers to be manipulating and/or tolerante lookups when mapping to idiomatic JSON

Reply:
`camelCase` also suits me better, especially after spending a lot of time in JS/TS land, but I think I'll stay with PascalCase for now and sleep on that to decide if I change that in future.